__tests__/app.test.js

```diff
@@ -115,7 +115,7 @@ test("GET 404: responds with an error when wrong endpoint is requested", () => {
 });
 
 describe("/api/articles/:article_id", () => {
-  test("GET 200: responds with correct article with all keys", () => {
+  test("GET 200: responds with correct article with all keys including a comment count equal to a sum of all the comments for that article", () => {
```

Comment:
🚨 Whenever you add a feature onto your app you don't want to backdate your tests. you should be making a new test and just checking the comment_count is also there. By going back and changing the test you're making a lot of work for yourself